package com.example.j2n.exception;

public class DataNotFoundException extends BaseServiceException {
    public DataNotFoundException(String message) {
        super(message);
    }

    public DataNotFoundException(String message, Throwable cause) {
        super(message, cause);
    }
}
